### PR TITLE
Deploy assets

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: en/developer_guide/deploy_assets.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+deploy asset
```
Comment (alavieille) [inline en/developer_guide/deploy_assets.rst]: Deploy

File: en/developer_guide/deploy_assets.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+============
+
+Context
+-------
+
+As specified in Symfony Reference Documents :
+
+The assets version "is used to bust the cache on assets by globally adding a query parameter to all rendered asset paths (e.g. /images/logo.png?v2).
```
Comment (alavieille) [inline en/developer_guide/deploy_assets.rst]: indent the paragraph to create a block quote

**Comment by alavieille** [comment]
May be add a paragraph to explain the task capistrano in this case where the developer uses capistrano for deploy her application.

File: en/developer_guide/deploy_assets.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+.. code-block:: yaml
+
+    framework:
+        assets:
+            version: 1.0.0
+
+This assets version is used by Open Orchestra Back Office.
+In order to minimize downloads, the different templates used in the rendering of Back Office are stocked in browser's local storage.
+These templates have two indexes in local storage : assets version and Back Office language.
+So, If a template is needed during navigation, the browser checks if it is present in local storage regarding to current asset version and language of the Back Office.
```
Comment (nicolasThal) [inline en/developer_guide/deploy_assets.rst]: While rendering, the Back Office requires some underscore template. When asked, the browser will first check in the local storage, with the current asset version and current Back Office language as key. If the the template is found, there will be no request to the server to load it.

File: en/developer_guide/deploy_assets.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+In such case, the browser loads the template from local storage instead of loading from server.
+
+For this reason, in case of template updates, it is needed to change assets version in app/config/config.yml otherwise the changes will not appear.
```
Comment (nicolasThal) [inline en/developer_guide/deploy_assets.rst]: a modification of the assets version in the `app/config/config.yml` file is required

Comment (nicolasThal) [inline en/developer_guide/deploy_assets.rst]: Or : This process will require a modification of the asset version parameter in the `app/config/config.yml` file to allow the changes to appear.

File: en/developer_guide/deploy_assets.rst (hunk 3 above)
Comment (ngilain) [inline en/developer_guide/deploy_assets.rst]: So, if a template

**Comment by ngilain** [comment]
:+1:

**Comment by veolys-nanne** [comment]
done @itkg-alavieille , @nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**